public/js/layout.js

```diff
@@ -351,8 +351,34 @@ function layoutNode(x, y, id, label = "", type, parent) {
     this.prev = 'a';
     this.count = 0;
     this.parent = parent;
+    this.objectType = "Layout Node";    
+}
 
```

Comment:
The next comment explains the reason behind the "undefined" bug in layout mode left side bar and how to properly fix it.